cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Do we need this file? It's all commented out.

Reply:
Is generated by Cypress and imported by other files so decided to keep in case I add any code to it in the future.